tests/integration/test_npm.py

```diff
@@ -77,6 +77,30 @@
             ),
             id="npm_lockfile3_yarn_registry",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-multiple-packages.git",
+                ref="b37466c6a680bb82e087c17ac0232b554f7f2cc6",
+                packages=(
+                    {"path": "first_pkg", "type": "npm"},
+                    {"path": "second_pkg", "type": "npm"},
+                ),
+                check_vendor_checksums=False,
+            ),
+            id="npm_multiple_packages_lockfile1",
```

Comment:
We might drop support for V1. See my PR https://github.com/containerbuildsystem/cachi2/pull/229

Reply:
Please provide your feedback on dropping V1.